rust/src/smb/smb2.rs

```diff
@@ -449,6 +501,16 @@ pub fn smb2_request_record<'b>(state: &mut SMBState, r: &Smb2Record<'b>)
                             tx.vercmd.set_smb2_cmd(SMB2_COMMAND_SET_INFO);
                             true
                         }
+                        Smb2SetInfoRequestData::EOF(ref eof) => {
+                            if let Some(_fname) = state.guid2name_map.get(rd.guid) {
+                                // gets removed on file close
+                                //TODOsmbmulti5 by the way guid2name_map never removed and not limited !
```

Comment:
Should I create a ticket for `guid2name_map` : stateful structure forever growing without limits ?

Reply:
Yes. We need a review of all the HashMaps in SMB and NFS state